repos/system_upgrade/common/actors/checkyumpluginsenabled/libraries/checkyumpluginsenabled.py

```diff
@@ -0,0 +1,50 @@
+from leapp import reporting
+from leapp.libraries.common.rhsm import skip_rhsm
+
+REQUIRED_YUM_PLUGINS = ['subscription-manager', 'product-id']
+FMT_LIST_SEPARATOR = '\n    - '
+
+
+def check_required_yum_plugins_enabled(yum_config):
+    """
+    Checks whether the yum plugins required by the IPU are enabled.
+
+    If they are not enabled, a report is produced informing the user about it.
+    Does not inhibit the IPU.
+
+    :param yum_config: YumConfig
+    """
+
+    if skip_rhsm():
+        return
+
+    missing_required_plugins = set(REQUIRED_YUM_PLUGINS) - set(yum_config.enabled_plugins)
```

Comment:
In case you apply the comment about making `REQUIRED_YUM_PLUGINS` a set already.
```suggestion
    missing_required_plugins = REQUIRED_YUM_PLUGINS - set(yum_config.enabled_plugins)
```

Reply:
Applied in the latest version. Thank you.